eng/ci/templates/official/jobs/build-artifacts-linux.yml

```diff
@@ -26,7 +26,12 @@ jobs:
     os: linux
 
   steps:
-  - template: /eng/ci/templates/install-dotnet.yml@self
+  # TODO: revert to global.json when Windows is unblocked from moving forward.
```

Comment:
This would also remove the installs stated as needed for test, etc. Without knowing the full usage path for those, I'm wondering if we should let them be and just add in version 8.x . Do we need to remove the template?

Reply:
The tests ran in a different job which still references the template